docker-compose.template

```diff
@@ -15,6 +22,7 @@ hydroshare:
     POSTGIS_DB: postgres
     POSTGIS_USER: postgres
     PGPASSWORD: postgres
+    SOLR_HOST: hydroshare_solr_1
```

Comment:
it is using the container name, this is just providing a way to override that if need be.